src/charm.py

```diff
@@ -602,6 +602,8 @@ def _current_effective_db_info(self) -> Optional[types_.DatabaseConfig]:
         host, port = self._parse_database_endpoints(
             self.database.fetch_relation_field(relation.id, "endpoints")
         )
+        if host is None:
+            return None
```

Comment:
What about adding some log indicating that the relation is there but the endpoint (for some reason?) is not available?

Reply:
Yes, indeed, updated! But I suspect anyone will look the log if there's no errors 😂

Reply:
You'd be surprised. Quite often when things don't happen as expected even if there's no error you end up looking in the logs. Having logging like this is definitely very useful.